### PR TITLE
Added deployments for Prater Jato V2

### PR DESCRIPTION
**Pull request by systemblox**

### CODE REVIEW

File: .k8/prater/ssv-node-prater-1-deployment.yml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+          - name: LOG_LEVEL
+            value: "debug"
+          - name: DB_REPORTING
+            value: "false"
+          - name: PUBSUB_TRACE
+            value: "false"
+          - name: DISCOVERY_TYPE_KEY
+            value: "discv5"
+          - name: NETWORK
+            value: "jato-v2-stage"
```
Comment (olegshmuelov) [inline .k8/prater/ssv-node-prater-1-deployment.yml:96]: @moshe-blox 
If I understand right, its related to network config
If so we have to create a config for jato-v2-prater-prod

File: .k8/prater/ssv-node-prater-1-deployment.yml [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+          - name: GENESIS_EPOCH
+            value: "1"
+          - name: DB_PATH
+            value: "./data/db-jato-v2"
+          - name: METRICS_API_PORT
+            value: "15011"
+          - name: ENABLE_PROFILE
+            value: "true"
+          - name: NETWORK_ID
+            value: "0x00003012"
```
Comment (olegshmuelov) [inline .k8/prater/ssv-node-prater-1-deployment.yml:112]: Could be the network id should be handled in the network config as well

**Review by olegshmuelov — CHANGES_REQUESTED** [review]
We have to remove the old deployment yamls

File: networkconfig/jato-v2.go [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+var JatoV2 = NetworkConfig{
+	Name:         "jato-v2",
+	Beacon:       beacon.NewNetwork(spectypes.PraterNetwork),
+	Domain:       spectypes.V3Testnet,
+	GenesisEpoch: 205852,
+
+	// TODO: should be adjusted to the actual offset once we have a contract
+	ETH1SyncOffset: new(big.Int).SetInt64(9015219),
+	// TODO: should be adjusted to the actual contract address once we have a contract
+	RegistryContractAddr: "0x01",
```
Comment (y0sher) [inline networkconfig/jato-v2.go:20]: lets merge after we have this

File: .gitlab-ci.yml [hunk 4]
```diff
@@ -122,9 +123,15 @@ Deploy v3 ssv-nodes to blox-infra-prod cluster:
...
+    # - .k8/scripts/deploy-ssv-node-v3-yamls-on-prod-k8s.sh $DOCKER_REPO_INFRA_PROD $CI_BUILD_REF ssv $APP_REPLICAS_INFRA_PROD blox-infra-prod kubernetes-admin@blox-infra-prod ssv.network $K8S_API_VERSION $PROD_HEALTH_CHECK_IMAGE $SSV_NODES_CPU_LIMIT_V3 $SSV_NODES_MEM_LIMIT_V3
+    #
+    # +------------------------+
+    # │  Deploy Prater Jato V2 |
+    # +------------------------+
+    - .k8/scripts/deploy-ssv-full-nodes-prater-jato-v2-yamls-on-stage-k8s.sh $DOCKER_REPO_INFRA_PROD $CI_BUILD_REF ssv $APP_REPLICAS_INFRA_PROD blox-infra-prod kubernetes-admin@blox-infra-prod ssv.network $K8S_API_VERSION $SSV_EXPORTER_CPU_LIMIT $SSV_EXPORTER_MEM_LIMIT
+    - .k8/scripts/deploy-ssv-nodes-prater-jato-v2-yamls-on-stage-k8s.sh $DOCKER_REPO_INFRA_PROD $CI_BUILD_REF ssv $APP_REPLICAS_INFRA_PROD blox-infra-prod kubernetes-admin@blox-infra-prod ssv.network $K8S_API_VERSION $PROD_HEALTH_CHECK_IMAGE $SSV_NODES_CPU_LIMIT_V3 $SSV_NODES_MEM_LIMIT_V3
```
Comment (y0sher) [inline .gitlab-ci.yml:132]: should this stay? maybe this can create the foundation to launching mainnet?

File: .gitlab-ci.yml [hunk 5]
```diff
@@ -106,7 +107,7 @@ Build prod Docker image:
     - docker tag $IMAGE_NAME:$CI_BUILD_REF $DOCKER_REPO_INFRA_PROD:$CI_BUILD_REF
     - $DOCKER_LOGIN_TO_INFRA_PROD_REPO && docker push $DOCKER_REPO_INFRA_PROD:$CI_BUILD_REF
   only:
-    - main
+    - stage-jato-v2-setup-ci-cd
```
Comment (lior-blox) [inline .gitlab-ci.yml:110]: need to change main

**Review by y0sher — APPROVED** [review]

File: networkconfig/jato-v2.go (hunk 3 above)
Comment (lior-blox) [reply networkconfig/jato-v2.go]: done

**Review by olegshmuelov — APPROVED** [review]

File: .k8/prater/ssv-node-prater-1-deployment.yml (hunk 2 above)
Comment (lior-blox) [reply .k8/prater/ssv-node-prater-1-deployment.yml]: done

**Pull request merged**